runtime/vm/createramclass.cpp

```diff
@@ -3108,6 +3108,13 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 			if (J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(className), J9UTF8_LENGTH(className), MAGIC_ACCESSOR_IMPL)) {
 				classFlags |= J9ClassIsExemptFromValidation;
 			}
+
+#if JAVA_SPEC_VERSION == 8
+			if (J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(className), 20, "com/sun/proxy/$Proxy")) {
```

Comment:
The hardcoded `20` doesn't seem right.  Looking at the definition of the macro it should be the `J9UTF8_LENGTH(className)`:

https://github.com/eclipse/openj9/blob/097492a09bfc0767aeea69141794c39063e080db/runtime/oti/j9.h#L174

If the intention is to check a substring, it would be clearer to use `strncmp` after validating the length and adding a comment to indicate why strncmp is correct